prowler/providers/aws/lib/service/service.py

```diff
@@ -101,3 +101,16 @@ def __threading_call__(self, call, iterator=None):
             except Exception:
                 # Handle exceptions if necessary
                 pass  # Replace 'pass' with any additional exception handling logic. Currently handled within the called function
+
+    def get_unknown_arn(self, region: str = None) -> str:
```

Comment:
I'd add here also `resource_id: str = unknown` since we will need it for other services.

Reply:
Done